cli/tests/integrations/test_package.py

```diff
@@ -683,7 +697,8 @@ def test_install_no():
                 b'catalog-terms-conditions/#community-services\n'
                 b'A sample pre-installation message\n'
                 b'Continue installing? [yes/no] Exiting installation.\n'
-            )
+            ),
+            returncode=1
```

Comment:
Hmm. At first I thought this seemed strange to return a `1` if the user supplied `no` as valid input. The reason being that the command exits cleanly, whether the user inputs `yes` or `no` (the only difference being whether the package was actually installed or not). I would have expected a `1` only if there was some internal error that made the command exit prematurely.

On thinking about it a bit more though, I agree with the semantics you have in place. Use a `0` to indicate that the command successfully carried out what you asked it to; use a `1` to indicate that it didn't.

An example demonstrating these semantics in a standard unix environment:
```
$ ls
foo        bar        baz
$ ls | grep foo
$ echo $?
0
$ ls | grep qux
$ echo $?
1
```